native-sql-engine/core/src/main/scala/com/intel/oap/execution/ColumnarWholeStageCodegenExec.scala

```diff
@@ -568,7 +592,7 @@ case class ColumnarWholeStageCodegenExec(child: SparkPlan)(val codegenStageId: I
       def close = {
         closed = true
         pipelineTime += (eval_elapse + build_elapse) / 1000000
-        buildRelationBatchHolder.foreach(_.close)
+        buildRelationBatchHolder.foreach(_.close) // fixing: ref cnt goes nagative
```

Comment:
this seems not necessary now as the iter will be closed when kernel close?

Reply:
It seems like BHJ & SHJ are still using this for caching data